docs/usage/shoot_settings/default_seccomp_profile.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Default Seccomp Profile
+description: Enable the use of `RuntimeDefault` as the default seccomp profile through `spec.kubernetes.kubelet.seccompDefault`
+---
+
+# Default Seccomp Profile and Configuration 
+
+This is a short guide describing how to enable the defaulting of seccomp profiles for Gardener managed workloads in the seed. Running pods in `Unconfined` (seccomp disabled) mode is undesirable since this is the least restrictive profile. Also, mind that any privileged container will always run as `Unconfined`. More information about seccomp can be found in this [Kubernetes tutorial](https://kubernetes.io/docs/tutorials/security/seccomp/).
+
+## Setting the Seccomp Profile to RuntimeDefault for Shoot Clusters
+
+You can enable the use of `RuntimeDefault` as the default seccomp profile for all workloads. If enabled, the kubelet will use the `RuntimeDefault` seccomp profile by default, which is defined by the container runtime, instead of using the `Unconfined` mode. More information for this feature can be found in the [Kubernetes documentation](https://kubernetes.io/docs/tutorials/security/seccomp/#enable-the-use-of-runtimedefault-as-the-default-seccomp-profile-for-all-workloads).
+
+To use seccomp profile defaulting, you must run the kubelet with the `SeccompDefault` feature gate enabled (this is the default).
```

Comment:
`(this is the default)` - this is not a good explanation. Enabled by default since which K8s version? What is the feature gate state across the different K8s versions?